### ISSUE TITLE
Acces denied (3.0.0_beta02)

### ISSUE THREAD

**Issue by necotec** [Kunzisoft/KeePassDX#1077]
**Description**
I use KeePassDX on a VollaPhone with Android 10. after a fdroid Update to KeePassDX Version 3.0.0 Beta2, i can not open my local Databases. There is an 'Access denied' exception. After downgrade back to Version 2.10 all things are ok.

**To Reproduce**
1. Update KeePassDX via fdroid to Version 3.0.0 beta2
2. Open the App and try to unlock your database
3. You should see the exeption on the display bottom.

**Expected behavior**

**KeePass Database**
 - Created with: Linux KeePass 2.42
 - Version: 2
 - Location: Nextcloud folder on local device
 - Size: 77kb
 - Contains attachment: No

**KeePassDX (please complete the following information):**
 - Version: 3.0.0beta2
 - Build: I don't know
 - Language: english

**Android (please complete the following information):**
 - Device: VollaPhone
 - Version: Android 10

**Comment by J-Jamet** [comment]
Very strange, the problem was present in version 3.0.0_beta01 but was fixed in version beta02, are you sure you looked at the version number in the pplication and not only on F-Droid after downgrading?

**Comment by J-Jamet** [comment]
Linked to #1066 #1067 #1071

**Comment by DrKittens** [comment]
Can confirm this fault was present I'd beta 3.0.0 as I experienced it.

After upgrading to Beta2 the fault remained until I reselected my keyfile.

In Beta1 the workaround to open the database I used was to reselect my keyfile each launch as keepass appeared to remember an incorrect (too correct) path for it.

I am no longer experiencing this issue or need to use the workaround after updating to 3.0.0beta2.

This can likely be closed. @necotec try re-selecting your keyfile then unlocking with biometrics/passphrase on 3.0.0beta2.
